locale/ja/translation.json

```
@@ -21,6 +21,7 @@
     "home": "Home",
     "archive-home": "Archived Docs Home",
     "archive-label": "Archived Docs",
+    "lts-label": "Latest LTS Docs",
```

Please use lower camel case.
```suggestion
    "ltsLabel": "Latest LTS Docs",
```